napari_tiff/napari_tiff_reader.py

```diff
@@ -299,7 +313,27 @@ def imagej_reader(tif):
         blending=blending,
         visible=visible,
     )
-    return [(data, kwargs, 'image')]
+    return kwargs
+
+
+def get_ome_tiff_metadata(tif):
+    metadata = xml2dict(tif.ome_metadata)
+    if 'OME' in metadata:
+        metadata = metadata['OME']
+
+    # TODO: process ome metadata
+
+    kwargs = dict(
+        rgb=rgb,
```

Comment:
It looks like these variables are not defined in this function? (`rgb`, `channel_axis`, `name`, `scale`, `colormap`, `contrast_limits`, `blending`, `visible`, etc.)